File: php_SDK/com/payhub/ws/model/RecurringBillResponse.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 24/07/2015
 * Time: 11:31
 */
class RecurringBillResponse
{
    public $recurringBillId;
    public $customerReference;
    public $billingReferences;

    /**
     * @return mixed
     */
    public function getRecurringBillId()
    {
        return $this->recurringBillId;
    }

    /**
     * @return mixed
     */
    public function getCustomerReference()
    {
        return $this->customerReference;
    }

    /**
     * @return mixed
     */
    public function getBillingReferences()
    {
        return $this->billingReferences;
    }


    public static function fromArray($data){
        if(!is_null($data)) {
            $recurringBill = new RecurringBillResponse();

            foreach ($data as $key => $value) {
                if (property_exists(get_class($recurringBill), $key)) {
                    if ($key == "billingReferences") {
                        $recurringBill->{$key} = BillingReferences::fromArray($value);
                    } elseif ($key == "customerReference") {
                        $recurringBill->{$key} = CustomerReference::fromArray($value);
                    } else {
                        $recurringBill->{$key} = $value;
                    }
                }
            }
            return $recurringBill;
        }
    }
}

File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/model/RecurringBillResponse.java
/**
 * 
 */
package com.payhub.ws.model;

import com.fasterxml.jackson.annotation.JsonInclude;

/**
 * @author agustin
 *
 */
@JsonInclude(JsonInclude.Include.NON_NULL) 
public class RecurringBillResponse {
	 private String recurringBillId;

	 private CustomerReference customerReference;

	 private BillingReferences billingReferences;

	public String getRecurringBillId() {
		return recurringBillId;
	}

	public void setRecurringBillId(String recurringBillId) {
		this.recurringBillId = recurringBillId;
	}

	public CustomerReference getCustomerReference() {
		return customerReference;
	}

	public void setCustomerReference(CustomerReference customerReference) {
		this.customerReference = customerReference;
	}

	public BillingReferences getBillingReferences() {
		return billingReferences;
	}

	public void setBillingReferences(BillingReferences billingReferences) {
		this.billingReferences = billingReferences;
	}
	 
	        
}
